Annex60/Fluid/BaseClasses/FlowModels/basicFlowFunction_dp.mo

```diff
@@ -75,20 +73,5 @@ Changed implementation to allow <code>k=0</code>, which is
 the case for a closed valve with no leakage
 </li>
 </ul>
-</html>",
-revisions="<html>
-<ul>
-<li>
-August 4, 2011, by Michael Wetter:<br/>
-Removed option to use a linear function. The linear implementation is now done
-in models that call this function. With the previous implementation,
-the symbolic processor may not rearrange the equations, which can lead
-to coupled equations instead of an explicit solution.
-</li>
-<li>
-April 13, 2009, by Michael Wetter:<br/>
```

Comment:
This part got lost in the fix, I will reintroduce it before accepting the pull request.